### PR TITLE
[us] remove dead code and unused imports

### PR DESCRIPTION
**Pull request by rullzer**
See: https://github.com/owncloud/core/pull/25763

CC: @LukasReschke @MorrisJobke @nickvergessen @icewind1991

### CODE REVIEW

**Comment by rullzer** [comment]
Aaah it seems the repo is moved... let see.

File: apps/dav/lib/AppInfo/Application.php [hunk 1]
```diff
@@ -112,23 +107,6 @@ public function __construct (array $urlParams=array()) {
 			);
 		});
 
-		$container->registerService('OCA\DAV\Migration\Classification', function ($c) {
```
Comment (nickvergessen) [inline apps/dav/lib/AppInfo/Application.php]: Did we also kill the update steps

Comment (nickvergessen) [inline apps/dav/lib/AppInfo/Application.php]: We still have them, so this should be kept

Comment (nickvergessen) [inline apps/dav/lib/AppInfo/Application.php]: https://github.com/owncloud/core/pull/25654/files - Not sure if we want to also kill them

Comment (rullzer) [inline apps/dav/lib/AppInfo/Application.php]: aaah right lets think about that.

**Comment by rullzer** [comment]
Ok lets close this and just go over it ourself when we touch the code.

**Pull request closed without merging**